config/clusters/leap/cluster.yaml

```
@@ -0,0 +1,29 @@
+name: pangeo-hubs
```

```suggestion
name: leap
```

I think this should be leap, right?

---

Ah yes! Fixed.